corehq/apps/case_search/fixtures.py

```diff
@@ -0,0 +1,61 @@
+from lxml.builder import E
+
+from casexml.apps.phone.fixtures import FixtureProvider
+
+from corehq.apps.case_search.exceptions import CaseFilterError
+from corehq.apps.case_search.filter_dsl import build_filter_from_xpath
+from corehq.apps.es.case_search import CaseSearchES
+from corehq.messaging.templating import (
+    MessagingTemplateRenderer,
+    NestedDictTemplateParam,
+)
+
+
+def _get_user_template_info(restore_user):
+    return {
+        "username": restore_user.username,
+        "uuid": restore_user.user_id,
+        "user_data": restore_user.user_session_data
+    }
+
+
+def _get_template_renderer(restore_user):
+    renderer = MessagingTemplateRenderer()
+    renderer.set_context_param('user', NestedDictTemplateParam(_get_user_template_info(restore_user)))
```

Comment:
This is where the context is provided for interpolating the CSQL template strings.  App builders will only be able to reference things we add here, which as of this writing is only username, ID, and user data